# Defines a message for holding the six distance values.
int32 distance_a
int32 distance_b
int32 distance_c
int32 distance_d
int32 distance_e
int32 distance_f
